test/public/runs/detail.test.js

```diff
@@ -527,4 +553,29 @@ module.exports = () => {
             innerText: 'CmCvjNbg',
         });
     });
+
+    it('should display correct tag styling after updating in tag overview', async () => {
+        let badges;
+        // Fetch the run data before update of tag
+        await goToRunDetails(page, 106);
+
+        badges = await getBadges();
+
+        expect(badges[0].backgroundColor === 'rgb(255, 0, 0)').to.be.false;
```

Comment:
If you always use `badges[0].backgroundColor`, you should have a function that simply return the background color of the first badge (YAGNI principle, the simplest is often the best). This function should be named accordingly.

Also, tests should be precises. int this case, you already know from the seeders what is the color of the badge, you should expect this color to be true (and not the badge to be `not red`)